### DISCUSSION TITLE
What does the "toggle overlay" does?

### DISCUSSION THREAD

**Discussion by ognistik** [cymian/mouseless#22] [Q&A]
I'm trying to set up my custom shortcuts, and one thing that I would like to do is to use the exact same key—which in my case is the left control—to toggle the overlay on and off. However, I don't seem to be able to do that. I can show the overlay with one key, hide the overlay with another key, but the `toggle overlay` setting doesn't seem to be working. Actually, whenever I try to set it to any key and click apply, nothing happens. I'm not even sure if there’s an error; nothing pops up, same when I try it with or without the overlay active. I just don’t see it working at all.

**Comment by cymian** [comment]
yep, couple bugs here, see #13

will fix soon
